worker.go

```diff
@@ -73,9 +73,7 @@ func (w *worker) process(message *Msg) (err error) {
 		}
 	}()
 
-	return w.manager.mids.call(w.manager.queueName(), message, func() error {
-		return w.manager.job(message)
-	})
+	return w.manager.job(message)
```

Comment:
I like the brevity of this line after you changes. But I am not sure if `job` is the correct name. If this file is read without context, the reader might not realize that the middlewares are also called. I would prefer a name that is more expressive of the fact that the job + middleware funcs are call chained here.